invisible_cities/cities/components_test.py

```diff
@@ -318,3 +319,14 @@ def test_check_max_time_units():
     with raises(ValueError):
         check_max_time(max_time, buffer_length)
 
+def test_read_wrong_pmt_ids(ICDATADIR):
+    """
+    Check if an error is raised when wrong PMT IDs are read from nexus file
+    """
+    file_in    = os.path.join(ICDATADIR, "nexus_next100_full_wrong_PMT_IDs.h5")
+    run_number = 0
+    rate       = 0.5
+
+    with raises(SensorIDMismatch):
+        sns_gen = mcsensors_from_file([file_in], 'next100', run_number, rate)
+        _ = next(sns_gen)
```

Comment:
```suggestion
    sns_gen = mcsensors_from_file([file_in], 'next100', run_number, rate)
    with raises(SensorIDMismatch):
        next(sns_gen)
```
to make it more clear where the exception is raised.